playbooks/roles/capsule/tasks/devel_install.yml

```diff
@@ -0,0 +1,19 @@
+- name: 'start rails server'
+  command: /home/vagrant/.rvm/bin/rvm ruby-2.2.4 do rails s
```

Comment:
It just exits saying "a server is already running" I am pretty sure I have had one running when I ran this playbook, but will test it again to be sure.